### ISSUE TITLE
Poor training results

### ISSUE THREAD

**Issue by RylonW** [yastrebksv/TennisCourtDetector#13]
> Epoch: 5
val loss = 0.21203158216939555
Val/precision0.004365 Val/accuracy0.004595
Epoch: 10
val loss = 0.20682167856813335
Val/precision0.0002610 Val/accuracy0.0004210
Epoch: 15
val loss = 0.20165815874398182
Val/precision0.0001915 Val/accuracy0.0003915
Epoch: 20
val loss = 0.19649156542133084
Val/precision0.000120 Val/accuracy0.0007120
Epoch: 25
val loss = 0.19153208773341968
Val/precision6e-0525 Val/accuracy0.0010325
Epoch: 30
val loss = 0.1866528452943555
Val/precision0.0001330 Val/accuracy0.0015230
Epoch: 35
val loss = 0.18181005270361042
Val/precision0.000135 Val/accuracy0.0015535
Epoch: 40
val loss = 0.17719185470248297
Val/precision0.000140 Val/accuracy0.00240
Epoch: 45
val loss = 0.1726313765100438
Val/precision0.000145 Val/accuracy0.0016845
Epoch: 50
val loss = 0.16808226372269416
Val/precision0.000150 Val/accuracy0.0022350
Epoch: 55
val loss = 0.16373939394093245
Val/precision0.000155 Val/accuracy0.0020455
Epoch: 60
val loss = 0.15940999566651076
Val/precision0.0001660 Val/accuracy0.0022360
Epoch: 65
val loss = 0.15513876327078976
Val/precision0.000165 Val/accuracy0.0022665
Epoch: 70
val loss = 0.15099892802804493
Val/precision6e-0570 Val/accuracy0.0019470
Epoch: 75
val loss = 0.14671144854250573
Val/precision0.0001375 Val/accuracy0.0022975
Epoch: 80
val loss = 0.14281168815900955
Val/precision0.000180 Val/accuracy0.0023680
Epoch: 85
val loss = 0.13892301531146756
Val/precision0.000185 Val/accuracy0.00285
Epoch: 90
val loss = 0.13501548906453223
Val/precision0.000190 Val/accuracy0.0022990
Epoch: 95
val loss = 0.13146534090419468
Val/precision3e-0595 Val/accuracy0.0023995
Epoch: 100
val loss = 0.127644996420085
Val/precision0.00013100 Val/accuracy0.00249100
Epoch: 105
val loss = 0.12399294784815192
Val/precision0.0001105 Val/accuracy0.002105
Epoch: 110
val loss = 0.12051866578755618
Val/precision0.00039110 Val/accuracy0.00265110
Epoch: 115
val loss = 0.11699845609690646
Val/precision0.00013115 Val/accuracy0.00239115
Epoch: 120
val loss = 0.11371946940533549
Val/precision0.00016120 Val/accuracy0.00262120
Epoch: 125
val loss = 0.11038124046737341
Val/precision0.00036125 Val/accuracy0.00297125
Epoch: 130
val loss = 0.1072519386200596
Val/precision0.00026130 Val/accuracy0.00258130
Epoch: 135
val loss = 0.10412519950446465
Val/precision0.00026135 Val/accuracy0.00246135

The above is my traing log. The loss is decreasing but the precision and accuracy are very low. I do not change any code. Could you please tell me why this happens? Thank you so much.
